### PR TITLE
Cloud-native book updates for .NET 6

### PR DESCRIPTION
**Pull request by robvet**
Please review and merge the contained updates for .NET 6.

Thanks.

### CODE REVIEW

File: docs/architecture/cloud-native/deploy-containers-azure.md [hunk 1]
```diff
@@ -93,34 +93,15 @@ This information is sufficient to get started. As part of the creation process i
 
 This [quickstart walks through deploying an AKS cluster using the Azure portal](/azure/aks/kubernetes-walkthrough-portal).
 
-## Azure Dev Spaces
+## Azure Bridge to Kubernetes
 
-Cloud-native applications can quickly grow large and complex, requiring significant compute resources to run. In these scenarios, the entire application can't be hosted on a development machine (especially a laptop). Azure Dev Spaces is designed to address this problem using AKS. It enables developers to work with a local version of their services while hosting the rest of the application in an AKS development cluster.
+Cloud-native applications can grow large and complex, requiring significant compute resources to run. In these scenarios, the entire application can't be hosted on a development machine (especially a laptop). [Azure Bridge to Kubernetes](https://docs.microsoft.com/visualstudio/bridge/overview-bridge-to-kubernetes) addresses the shortcoming. It enables developers to work with a local version of their service while hosting the entire application in an AKS development cluster.
```
Comment (IEvangelist) [inline docs/architecture/cloud-native/deploy-containers-azure.md:98]:
```suggestion
Cloud-native applications can grow large and complex, requiring significant compute resources to run. In these scenarios, the entire application can't be hosted on a development machine (especially a laptop). [Azure Bridge to Kubernetes](/visualstudio/bridge/overview-bridge-to-kubernetes) addresses the shortcoming. It enables developers to work with a local version of their service while hosting the entire application in an AKS development cluster.
```

File: s were configured automatically based [hunk 2]
```diff
@@ -0,0 +1,39 @@
+diff.astextplain.textconv=astextplain
```
Comment (IEvangelist) [inline s were configured automatically based:1]: I think this file should be deleted, @robvet

**Review by IEvangelist — APPROVED** [review]
This is awesome, great job on the updates - I have a few nits.

File: docs/architecture/cloud-native/application-resiliency-patterns.md [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
 
 The first line of defense is application resiliency.
 
-While you could invest considerable time writing your own resiliency framework, such products already exist. [Polly](https://dotnetfoundation.org/projects/polly) is a comprehensive .NET resilience and transient-fault-handling library that allows developers to express resiliency policies in a fluent and thread-safe manner. Polly targets applications built with either the .NET Framework or .NET 5. The following table describes the resiliency features, called `policies`, available in the Polly Library. They can be applied individually or grouped together.
+While you could invest considerable time writing your own resiliency framework, such products already exist. [Polly](https://dotnetfoundation.org/projects/polly) is a comprehensive .NET resilience and transient-fault-handling library that allows developers to express resiliency policies in a fluent and thread-safe manner. Polly targets applications built with either the legacy .NET Framework or .NET 6. The following table describes the resiliency features, called `policies`, available in the Polly Library. They can be applied individually or grouped together.
```
Comment (nishanil) [inline docs/architecture/cloud-native/application-resiliency-patterns.md:12]: I don't think we should call it .NET Framework "legacy"

**Review by nishanil — COMMENTED** [review]
Added a few more comments.

File: docs/architecture/cloud-native/deploy-containers-azure.md (hunk 1 above)
Comment (robvet) [reply docs/architecture/cloud-native/deploy-containers-azure.md:98]: Great catches -- thx!

File: s were configured automatically based (hunk 2 above)
Comment (robvet) [reply s were configured automatically based:1]: Yes, thx!

Comment (robvet) [reply s were configured automatically based:1]: Not seeing that line of code anywhere in repo. Please advise.  Better yet, if it's there, please delete it.

File: docs/architecture/cloud-native/application-resiliency-patterns.md (hunk 3 above)
Comment (robvet) [reply docs/architecture/cloud-native/application-resiliency-patterns.md:12]: Fixed

File: docs/architecture/cloud-native/grpc.md [hunk 4]
```diff
@@ -50,21 +50,21 @@ The book, [gRPC for WCF Developers](../grpc-for-wcf-developers/index.md), availa
 
 gRPC is integrated into .NET Core 3.0 SDK and later. The following tools support it:
 
-- Visual Studio 2019, version 16.3 or later, with the web development workload installed.
+- Visual Studio 2022 or later
```
[the remark this answers is not in the document]
Comment (robvet) [reply docs/architecture/cloud-native/grpc.md:53]: Fixed

File: docs/architecture/cloud-native/grpc.md [hunk 5]
```diff
@@ -50,21 +50,21 @@ The book, [gRPC for WCF Developers](../grpc-for-wcf-developers/index.md), availa
...
 - Visual Studio Code
 - the dotnet CLI
 
-The SDK includes tooling for endpoint routing, built-in IoC, and logging. The open-source Kestrel web server supports HTTP/2 connections. Figure 4-20 shows a Visual Studio 2019 template that scaffolds a skeleton project for a gRPC service. Note how .NET fully supports Windows, Linux, and macOS.
+The SDK includes tooling for endpoint routing, built-in IoC, and logging. The open-source Kestrel web server supports HTTP/2 connections. Figure 4-20 shows a Visual Studio 2022 template that scaffolds a skeleton project for a gRPC service. Note how .NET fully supports Windows, Linux, and macOS.
 
-![gRPC Support in Visual Studio 2019](./media/visual-studio-2019-grpc-template.png)
+![gRPC Support in Visual Studio 2022](./media/visual-studio-2022-grpc-template.png)
```
[the remark this answers is not in the document]
Comment (robvet) [reply docs/architecture/cloud-native/grpc.md:59]: Fixed

**Comment by robvet** [comment]
Great suggestions -- all implemented -- please review.

File: docs/architecture/cloud-native/grpc.md (hunk 4 above)
Comment (nishanil) [inline docs/architecture/cloud-native/grpc.md:53]:
```suggestion
- Visual Studio 2022 with the ASP.NET and web development workload installed.
```

File: docs/architecture/cloud-native/grpc.md [hunk 6]
```diff
@@ -50,21 +50,21 @@ The book, [gRPC for WCF Developers](../grpc-for-wcf-developers/index.md), availa
 
 gRPC is integrated into .NET Core 3.0 SDK and later. The following tools support it:
 
-- Visual Studio 2019, version 16.3 or later, with the web development workload installed.
+- Visual Studio 2022 with the ASP.NET and web development workload installed.
```
Comment (nishanil) [inline docs/architecture/cloud-native/grpc.md:53]:
```suggestion
- Visual Studio 2022 with the ASP.NET and web development workload installed
```

**Review by nishanil — APPROVED** [review]
Looks good to me! Thanks @robvet 
@IEvangelist Feel free to merge when you are ready.

File: s were configured automatically based (hunk 2 above)
Comment (nishanil) [reply s were configured automatically based]: I just deleted the file.

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**